woocommerce-gateway-ebanx/includes/class-wc-ebanx-gateway.php

```diff
@@ -836,6 +858,12 @@ protected function save_user_meta_fields($order)
 					update_user_meta($this->userId, '_ebanx_billing_chile_birth_date', sanitize_text_field($_POST['ebanx_billing_chile_birth_date']));
 				}
 			}
+
+			if (trim(strtolower($order->billing_country)) === WC_EBANX_Gateway_Utils::COUNTRY_COLOMBIA) {
```

Comment:
Please, use `$this->getTransactionAddress('country')` instead of `trim(strtolower($order->billing_country))`.